### ISSUE TITLE
Option 'valalgo' is unknown.

### ISSUE THREAD

**Issue by mpgn** [NotSoSecure/Blacklist3r#4]
Hello,

After compiling the tool in visual studio I don't have the same option as you :

From the readme:
![image](https://user-images.githubusercontent.com/5891788/70986127-993c6000-20bd-11ea-84a8-a0b309761bdf.png)

On my side:
![image](https://user-images.githubusercontent.com/5891788/70986047-7447ed00-20bd-11ea-95e7-2db7f5a17e94.png)

So of course, when I try with a sample command like 
```
`AspDotNetWrapper.exe --keypath MachineKeys.txt --encrypteddata 195A989biBjM_NAqqiie5DnHKfcwrNGDuT-Suumqmw6oVyLSsjCFx9Emhf034TDjcuC9mfwNbi6yD-1QlbhcUAgdTOwY0o0sNbg7bJrNyUEf6ZoyYh2QAZHhmxteN_cMQJI7C1WOBEl0ocihUVhKghdxegwRURcYx2h1uMbijX3jsEf59L8Uco_PpfFLN--RtcLTKUvtZd0fH5Sgc1JQmsvTBr7IJ4Ua01I8uyEPYNXZGYvssSzJ8YN6MXioky3WBXv9NGNxDpgTpIPWGetgZ0iOSaTmqPr6sPu4ndesUV4SKsBroIP6Y38rr8LwFCZBKDK5dli4kKwmy9xeM02qshCoLf8ppeOiK2aMLfb9jqkraoss2BflD3hpDdrYHVGH7ryTWQh4HABYDC7OOMgdld3WJ1CUfJ9pmr0qnVFD4Gc --decrypt --purpose=owin.cookie  --valalgo=hmacsha512 --decalgo=aes
```

I have this error:

```
ERROR(S):
  Option 'valalgo' is unknown.
  Option 'decalgo' is unknown.
  Option 'legacy' is unknown.
```

We use the same version, any idea why I didn't have all options available ?

**Comment by mpgn** [comment]
If this is related with the version of .net I have (>4.5) It's means we can use `valalgo` only on a old version of .net ?

**Comment by LukaSikic** [comment]
@mpgn did you manage to make it work? I am experiencing the same issue

**Comment by mpgn** [comment]
Use the release version https://github.com/NotSoSecure/Blacklist3r/releases

**Comment by devsecboy** [comment]
The issue is fixed in AspDotNetWrapper v3.0
